lib/active_model_serializers.rb

```diff
@@ -2,6 +2,7 @@
 require 'active_model/serializer/version'
 require 'active_model/serializer'
 require 'active_model/serializer/fieldset'
+require 'active_model/serializer/pagination'
 require 'active_model/serializable_resource'
```

Comment:
assuming we move the code into the json api adapter, this require should also move